guides/user/pipeline/expressions/index.md

```diff
@@ -138,16 +140,17 @@ context.
 To specify a property file:
 
 1. Archive it in your Jenkins job.
-2. Specify the name of the file in the pipeline configuration. Note that this
-    is just the name of the file, not the absolute path —
+2. Specify the name of the file in the pipeline configuration. 
+  > __Note:__ This is just the name of the file, not the absolute path —
     `mypropertyfile.properties` rather than
     `workspace/build/mypropertyfile.properties`.
-    ![](images/property-file.png)
+    
+![](images/property-file.png)
```

Comment:
Indent this one so that the first character (!) lines up under the first underscore character of the Note. This should cause the image to indent nicely under the note.